### ISSUE TITLE
[request] ability to turn off logs completely

### ISSUE THREAD

**Issue by atesin** [gabe565/CastSponsorSkip#119]
i suffer for this issue: #54 

i tried castspornsorskip for just 1 day, then my home server crashes, i hardly went to check and realized my disk has filled up with log messages like:

`2023/12/11 21:44:33 ERROR Failed to discover devices. error="unable to create new zeroconf resolver: listen udp6 [ff02::]:5353: socket: address family not supported by protocol"`

messages log so often than even with 1sec running, screen fulls and i cant scroll up to beginning with `journalctl -fe-u castspornsorskip`

i would like a way to suppress at all or filter or redirect log messages, as the lowest log level is `error` and these are precisely the level of these messages so cant be skipped

**Comment by gabe565** [comment]
@atesin Hmm you're right that it would be nice to suppress these messages. I should be able to add a `none` log level which would disable all logs! Let me investigate.

**Comment by gabe565** [comment]
@atesin Ok, I added a `none` log level which will be in v0.8.1, but I believe the zeroconf error will also be resolved in this release!

**Comment by atesin** [comment]
> @atesin Hmm you're right that it would be nice to suppress these messages. I should be able to add a `none` log level which would disable all logs! Let me investigate.

that can alredy be done in systemd unit file: `StandardError=null` ... i wished a log filter option according a pattern, like rsyslog did.... too bad it seems that currently systemd doesnt work with rsyslog :/

**Comment by atesin** [comment]
OR...

raise severity of these messages, so it can be filtered when using `error` level severity

### OR (better)

add some other even lower severity level like `info`, so when we set this log level, `error` messages will be skipped

**Comment by gabe565** [comment]
Yeah I agree. I'm not a fan of lowering the log level below `error`, but the issue you were having sounded pretty frustrating. Is it fixed now?

**Comment by atesin** [comment]
honestly i gave up, just enabled ipv6 on my server although i dont use it in my lan

**Comment by gabe565** [comment]
@atesin The IPv6 issue should be fixed, but I haven't received any feedback yet. If you don't mind disabling IPv6 and letting me know if it's resolved in #54, then I could close that issue!
